server/routes/api/v1/physicians/update.js

```diff
@@ -0,0 +1,65 @@
+import { Role } from '../../../../models/user.js';
+import { StatusCodes } from 'http-status-codes';
+
+export default async function (fastify, _opts) {
+  fastify.patch(
+    '/:id',
+    {
+      schema: {
+        params: {
+          type: 'object',
+          properties: {
+            id: { type: 'string', format: 'uuid' },
+          },
+        },
+      },
+      response: {
+        [StatusCodes.OK]: {
+          type: 'object',
+          properties: {
+            id: { type: 'string' },
+            firstName: { type: 'string' },
+            middleName: { type: 'string' },
+            lastName: { type: 'string' },
+            phone: { type: 'string' },
+            email: { type: 'string' }
+          },
+        },
+        [StatusCodes.NOT_FOUND]: {
+          type: 'object',
+          properties: {
+            message: { type: 'string' },
+          },
+        },
+      },
+      onRequest: fastify.requireUser([
+        Role.ADMIN,
+        Role.STAFF,
+        Role.VOLUNTEER,
+        Role.FIRST_RESPONDER,
```

Comment:
First Responders will generally only have read access to data, I think we can remove it from the update route.